core/common/src/test/java/alluxio/security/login/LoginModuleTest.java

```diff
@@ -63,9 +64,11 @@ public void simpleLogin() throws Exception {
     loginContext.logout();
     assertTrue(subject.getPrincipals(User.class).isEmpty());
 
-    // logout twice should be no-op.
-    loginContext.logout();
-    assertTrue(subject.getPrincipals(User.class).isEmpty());
+    // logout twice should be no-op on Java 8
```

Comment:
why is it not in java 11?

Reply:
They changed the internal API so that logging out twice results in an NPE